src/containers/simulator/cell.js

```diff
@@ -1,20 +1,21 @@
 import React from 'react'
+import { corewar } from 'corewar';
 
 const Cell = props => {
 
   let css = '';
 
-  switch(props.cell.accessType) {
-    case 0:
-    css = 'read';
-    case 1:
-    css = 'write';
-    case 2:
-    css = 'execute'
-  }
+  // switch(props.cell.accessType) {
+  //   case 0:
+  //   css = 'read';
+  //   case 1:
+  //   css = 'write';
+  //   case 2:
```

Comment:
Might as well just delete this?

Reply:
Yeah, this is what I was confused about before. Was a cell supposed to show core + r/w/e + warriors etc? I was all mixed up.
I think as a proof of concept I'm gonna render 3 grids initially, one with each thing. Confirm they are working as expected and then go from there,
As a side point, a lot of what I've done thus far is messy (soz!) just trying to get a better understanding before I can produce some better quality codes. 

:shipit: